### PR TITLE
Fix item text alignment

### PR DESCRIPTION
**Pull request by felixheidecke**
Fixes https://github.com/owncloud/owncloud-design-system/issues/148

### CODE REVIEW

File: src/patterns/OcApplicationMenu.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@ import UiKit from "uikit"
  * ## ToDo:
  *
  * - [ ] Fix documentation of this component
- * - [ ] Fix item text alignment
+ * - [x] Fix item text alignment
```
Comment (DeepDiver1975) [inline src/patterns/OcApplicationMenu.vue]: feel free to just remove this line once fixed - THX a lot

**Comment by DeepDiver1975** [comment]
looks good - still WIP?

**Comment by felixheidecke** [comment]
oc menu is still in progress

**Comment by DeepDiver1975** [comment]
fixes #148

File: src/patterns/OcMenu.vue [hunk 2]
```diff
@@ -34,37 +33,35 @@ export default {
...
+<div>
+  <div class="uk-margin">
+    <oc-menu buttonText="Click Me!">
+      Welcome to ownCloud Design System!
+    </oc-menu>
+  </div>
+  <div class="uk-margin">
+    <oc-menu buttonText="Put &lt;oc-menu-item&gt; into the 'subnav' slot ...">
+      <template slot="subnav">
+        <oc-menu-item icon="create_new_folder" content="New folder …" />
```
Comment (DeepDiver1975) [inline src/patterns/OcMenu.vue]: while I see this: the text/content needs to be put into the inner tag - see #124 and others -> will open a new ticket

**Comment by DeepDiver1975** [comment]
@felixheidecke ready to review? please remove WIP - THX

**Pull request merged**